src/Link.php

```diff
@@ -654,7 +654,7 @@ public static function showAllLinksForItem(CommonDBTM $item, ?string $restrict_t
      * @since 0.85
      *
      * @param CommonDBTM $item The item
-     * @param array{id: int, name: string, link: string, data: string, open_window: ?bool} $params
+     * @param array{id?: int, name?: string, link?: string, data?: string, open_window?: ?bool} $params
```

Comment:
This doesn't seem right. The two places this is called both use `Link::getLinksDataForItem` which returns all required information directly from a DB query. At most maybe they need to be nullable, not optional.